### ISSUE TITLE
Swift Package usage: Unable to build iOS targets

### ISSUE THREAD

**Issue by yanniks** [apple/swift-openapi-generator#124]
Hi,

we are currently trying to migrate to this OpenAPI generator plugin. However, we noticed that builds of our Swift Package are no longer possible if the plugin is added to the target and an iOS Simulator is selected as target environment.

The following snippet is used to declare the target in the `Package.swift` file:
```
.target(
    name: "MyOAPIExample",
    dependencies: [
        .product(name: "OpenAPIRuntime", package: "swift-openapi-runtime"),
        "MyNetworkingLibrary",
    ],
    plugins: [
        .plugin(
            name: "OpenAPIGenerator",
            package: "swift-openapi-generator"
        ),
    ]
),
```

As soon as the plugin is added to the target, the build fails with the following issue:
```
error: The package product 'OpenAPIGenerator' requires minimum platform version 99.0 for the iOS platform, but this target supports 15.0 (in target 'MyOAPIExample' from project 'MyNetworking')
```

Builds who target macOS work fine. Is there any workaround that enables iOS builds?

Thanks & best regards,
Yannik

**Comment by czechboy0** [comment]
Hi @yanniks – I have a suspicion we might have regressed this in 0.1.4. Can you try to explicitly use version 0.1.3 of the generator as your dependency, and confirm that it works there?

**Comment by yanniks** [comment]
Hi @czechboy0 , thanks for the quick response! I can confirm that this issue is related to the 0.1.4 release.

**Comment by yanniks** [comment]
@czechboy0 issue is resolved in 0.1.5. Thanks for the quick release! 😊
